ksql-functional-tests/src/test/resources/query-validation-tests/key-field.json

```diff
@@ -129,45 +128,16 @@
       ],
       "post": {
         "sources": [
-          {"name": "INPUT", "type": "stream", "keyField": {"name": null, "legacyName": null}},
+          {"name": "INPUT", "type": "stream", "keyField": null},
           {
             "name": "OUTPUT",
             "type": "table",
-            "keyField": {"name": "FOO", "legacyName": "KSQL_INTERNAL_COL_0", "legacySchema": "STRING"},
+            "keyField": "FOO",
             "valueSchema": "STRUCT<FOO INT, KSQL_COL_1 BIGINT>"
           }
         ]
       }
     },
-    {
-      "name": "stream | initially null | group by (-) | key in value | no aliasing | legacy downstream query",
```

Comment:
Removed these legacy downstream queries as the property they use no longer exists